package.json

```diff
@@ -226,7 +226,6 @@
     "@kbn/eslint-plugin-license-header": "link:packages/kbn-eslint-plugin-license-header",
     "@kbn/plugin-generator": "link:packages/kbn-plugin-generator",
     "@kbn/test": "link:packages/kbn-test",
-    "@types/babel-core": "^6.25.5",
```

Comment:
__question:__ is that change somehow related to `makelogs`? Just asking, if we can remove that then it's great! :)

Reply:
Ohh sorry was going to write a comment about that. Yarn throw this out because it's an exact duplicate of 2 lines below.

Reply:
Ah, haven't noticed that :+1: